docs/api-reference/gltf-loaders/glb-builder.md

```diff
@@ -0,0 +1,79 @@
+# `GLBBuilder` class (@loaders.gl/gltf)
+
+The `GLBBuilder` class allows applications to use a "fluent" API to dynamically build up a hybrid JSON/binary GLB file. The `GLBBuilder` would normally be used if you want to save custom mixed JSON/binary data in a "GLB envelope".
+
+References:
+* For more information, see [glTF and GLB support]() in the Developer's Guide.
+
+
+## Usage
+
+Adding binary data sub chunks to the GLB file, then calling encode to generate the complete `arrayBuffer`.
+
+```js
+import {GLBBuilder} from '@loaders.gl/gltf';
+import {saveBinaryFile} from '@loaders.gl/core';
+
+const gltfBuilder = new GLBBuilder();
+
+const imageIndex = gltfBuilder.addImage();
```

Comment:
I do not understand what this is doing for the example.  It's not used and there is no "image" being added to the builder so it is a bit confusing